client/platform/desktop/backend/serializers/nist.ts

```diff
@@ -0,0 +1,393 @@
+
+import { MultiTrackRecord } from 'dive-common/apispec';
+import fs from 'fs-extra';
+import { AnnotationFileData } from 'platform/desktop/backend/serializers/viame';
+
+
+import {
+  TrackData, Feature, StringKeyObject,
+} from 'vue-media-annotator/track';
+import { RectBounds } from 'vue-media-annotator/utils';
+
+
+interface TrackJSON {
+  begin: number;
+  end: number;
+  trackId: number;
+  meta: StringKeyObject;
+  attributes: StringKeyObject;
+  confidencePairs: [string, number][];
+  features: Feature[];
+
+}
+
+type ActivityLocalization = Record<string, Record<string, number>>;
+// { filename: '123':0, '234':1} object with key equal to
+// filename and two number key with one equal to 0 and other 1
+type ObjectLocalization = Record<string,
+    Record<string,
+        Record<'boundingBox', {
+            x: number;
+            y: number;
+            w: number;
+            h: number;
+        }>
+    >
+>;
+
+export interface NistFile {
+    filesProcessed: string[];
+    processingReport: {
+        fileStatuses: Record<string, {
+            status: 'success' | 'fail';
+            message: string;
+        }>;
+        siteSpecific?: StringKeyObject;
+    };
+    activities: NistActivity[];
+
+}
+interface NistObject {
+    objectType: string;
+    objectID: number;
+    localization: ObjectLocalization;
+}
+
+interface NistActivity {
+    presenceConf: number;
+    alertFrame?: number;
+    activity: string;
+    localization: ActivityLocalization;
+    activityID: number;
+    objects?: NistObject[];
+}
+
+function confirmNistFormat(data: NistFile) {
+  return data.activities && data.filesProcessed;
+}
+
+async function confirmNistFile(filename: string) {
```

Comment:
return value of this function is weird.  Callers all treat it like a boolean.  Could the return type be a proper bool rather than `string[] | { tracks: never[]; }`?